### DISCUSSION TITLE
ModuleNotFoundError: No module named 'mindsdb.predictors'

### DISCUSSION THREAD

**Discussion by sbl57** [mindsdb/mindsdb#5652] [General]
Hi anybody

I am new to mindsdb, I have install it  with no problem but when I try to run a file test.py (which include  mdb = mindsdb.Predictor(name='combinations_predictor') ). I get the following error

python test.py 
Traceback (most recent call last):
  File "test.py", line 3, in <module>
    from mindsdb.predictors import Predictor
ModuleNotFoundError: No module named 'mindsdb.predictors'

Can anyone help to solve this issue?

**Comment by ZoranPandovski** [comment]
Hi, can you please provide more information about what you want to achieve? Directly importing the Predictor class is not the right way to interact with MindsDB. Please check the [docs](https://docs.mindsdb.com/sql/create/predictor) for more info
